extensions/oidc-client-reactive-filter/runtime/src/main/java/io/quarkus/oidc/client/reactive/filter/OidcClientRequestReactiveFilter.java

```diff
@@ -0,0 +1,56 @@
+package io.quarkus.oidc.client.reactive.filter;
+
+import java.util.function.Consumer;
+
+import javax.annotation.Priority;
+import javax.ws.rs.Priorities;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.Provider;
+
+import org.jboss.logging.Logger;
+import org.jboss.resteasy.reactive.client.spi.ResteasyReactiveClientRequestContext;
+import org.jboss.resteasy.reactive.client.spi.ResteasyReactiveClientRequestFilter;
+
+import io.quarkus.oidc.client.Tokens;
+import io.quarkus.oidc.client.runtime.AbstractTokensProducer;
+import io.quarkus.oidc.client.runtime.DisabledOidcClientException;
+import io.quarkus.oidc.common.runtime.OidcConstants;
+
+@Provider
+@Priority(Priorities.AUTHENTICATION)
+public class OidcClientRequestReactiveFilter extends AbstractTokensProducer implements ResteasyReactiveClientRequestFilter {
+    private static final Logger LOG = Logger.getLogger(OidcClientRequestReactiveFilter.class);
+    private static final String BEARER_SCHEME_WITH_SPACE = OidcConstants.BEARER_SCHEME + " ";
+
+    protected void initTokens() {
+        if (earlyTokenAcquisition) {
+            LOG.debug("Token acquisition will be delayed until this filter is executed to avoid blocking an IO thread");
```

Comment:
👍🏼

Reply:
Perhaps also mentione explicitly that the config option that will be ignored to make it actionable for the user?

Reply:
I was hoping you'd like this debug message :-)